telegram/passport/passportdata.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
```

Comment:
Regarding the docstrings here. As far as I can see it currently decrypts everything when comming in. So docstrings are off...